admin/src/main/scala/net/wiringbits/components/widgets/TableList.scala

```diff
@@ -0,0 +1,76 @@
+package net.wiringbits.components.widgets
+
+import com.alexitc.materialui.facade.materialUiCore.{components => mui}
+import com.alexitc.materialui.facade.materialUiCore.createMuiThemeMod.Theme
+import com.alexitc.materialui.facade.materialUiStyles.makeStylesMod.StylesHook
+import com.alexitc.materialui.facade.materialUiStyles.mod.makeStyles
+import com.alexitc.materialui.facade.materialUiStyles.withStylesMod.{
+  CSSProperties,
+  StyleRulesCallback,
+  Styles,
+  WithStylesOptions
+}
+import net.wiringbits.AppStrings
+import net.wiringbits.api.models.AdminGetTablesResponse
+import net.wiringbits.ui.components.core.widgets.{Container, Subtitle}
+import org.scalablytyped.runtime.StringDictionary
+import org.scalajs.dom
+import slinky.core.FunctionalComponent
+import slinky.core.facade.Fragment
+import slinky.web.html.className
+
+object TableList {
+  case class Props(response: AdminGetTablesResponse)
+
+  private lazy val useStyles: StylesHook[Styles[Theme, Unit, String]] = {
+    val stylesCallback: StyleRulesCallback[Theme, Unit, String] = theme =>
+      StringDictionary(
+        "tablesList" -> CSSProperties()
+          .setWidth("100%"),
+        "tableItem" -> CSSProperties()
+      )
+    makeStyles(stylesCallback, WithStylesOptions())
+  }
+
+  val component: FunctionalComponent[Props] = FunctionalComponent[Props] { props =>
+    val classes = useStyles(())
+
+    def goTo(table_name: String): Unit = {
+      val url = s"/tables/$table_name"
+      dom.window.location.href = url
```

Comment:
In react, you don't use these redirects because it reloads the whole page, instead, you use the react-router, like:

```
val history = reactRouterDom.useHistory()
history.push("/dashboard")
```

[example](https://github.com/wiringbits/scala-webapp-template/blob/master/web/src/main/scala/net/wiringbits/components/widgets/SignInForm.scala#L79)